### ISSUE TITLE
Sporadic SSLErrors on Client close

### ISSUE THREAD

**Issue by bmario** [metricq/metricq-python#18]
When stopping a client, we often get the following error:

```
2020-07-09 17:57:25,021 [ERROR   ] [asyncio             ] SSL error in data received
protocol: <asyncio.sslproto.SSLProtocol object at 0x7f3ebb2361d0>
transport: <_SelectorSocketTransport fd=7 read=polling write=<idle, bufsize=0>>
Traceback (most recent call last):
  File "/usr/lib/python3.7/asyncio/sslproto.py", line 526, in data_received
    ssldata, appdata = self._sslpipe.feed_ssldata(data)
  File "/usr/lib/python3.7/asyncio/sslproto.py", line 207, in feed_ssldata
    self._sslobj.unwrap()
  File "/usr/lib/python3.7/ssl.py", line 767, in unwrap
    return self._sslobj.shutdown()
ssl.SSLError: [SSL: KRB5_S_INIT] application data after close notify (_ssl.c:2609)
``` 

Good in reproducing the issue is the new metricq-spy tool.

**Comment by Zulan** [comment]
I vaguely recall that I had this symptom before adding a missing `await client.stop()` on the `HistoryClient` example. But `metricq-spy` seems to `await stop()` correctly already.

https://github.com/metricq/metricq-python/commit/8677fa353ca50e7fd5fc82903414626b0741cd1d

**Comment by tilsche** [comment]
I vaguely recall that I had this symptom before adding a missing `await client.stop()` on the `HistoryClient` example. But `metricq-spy` seems to `await stop()` correctly already.

https://github.com/metricq/metricq-python/commit/8677fa353ca50e7fd5fc82903414626b0741cd1d

**Comment by tilsche** [comment]
Is this related to https://github.com/mosquito/aio-pika/pull/344 ?

**Comment by kinnarr** [comment]
Python has an open issue to ignore this error after connection close: https://bugs.python.org/issue39951

**Comment by bmario** [comment]
Fixed in Python update.

**Comment by tilsche** [comment]
Do we know which exact releases this was fixed in? I understand it was not and will not be fixed in 3.7.

**Comment by bmario** [comment]
IIRC, it was fixed in a patch version of 3.7, but don't know for sure. haven't seen it in 3.8 for a while though. 🤷
